### PR TITLE
docs(PPDSC-1903): Select page

### PR DESCRIPTION
**Pull request by Srinivasan-Ramasamy**
PPDSC-1903

**What**

1. Created doc select page.

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on: 
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-1903-select-page 
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-1903-select-page/storybook

**Review by baburay23 — APPROVED** [review]

File: site/pages/components/select.tsx [hunk 1]
```diff
@@ -1,51 +1,57 @@
...
-  {
-    name: 'state',
-    type: ['valid', 'invalid', 'disabled'],
-    description:
-      'If provided, renders the FormInput Select in a valid, invalid, or disabled state. It can be submitted within a form.',
+    description: `Defines the size of the Select.`,
   },
   {
     name: 'useModal',
+    type: ['boolean'],
```
Comment (mutebg) [inline site/pages/components/select.tsx:26]: this is  `MQ<boolean>`

File: site/pages/components/select.tsx [hunk 2]
```diff
@@ -209,27 +206,42 @@ const selectOptionOverrides = [
       'If provided, this overrides the inset space within the Select option.',
   },
   {
-    attribute: 'spaceInline',
+    attribute: 'selectOption.spaceInline',
```
Comment (mutebg) [inline site/pages/components/select.tsx:209]: sorry but this is not correct, I guess the documentation in confluence is wrong

File: site/pages/components/select.tsx [hunk 3]
```diff
@@ -209,27 +206,42 @@ const selectOptionOverrides = [
...
+    description:
+      'If provided, this overrides the stylePreset of the Select option icon.',
+  },
+  {
+    attribute: 'selectOption.icon.size',
+    type: 'MQ<string>',
+    default: 'iconSize020',
+    description:
+      'If provided, this overrides the icon size of the Select option icon.',
+  },
```
Comment (mutebg) [inline site/pages/components/select.tsx:228]: this is also not correct

File: site/pages/components/select.tsx (hunk 2 above)
Comment (mutebg) [reply site/pages/components/select.tsx:209]: there are 2 components: Select and SelectOption and they have their own overrides. I think the part with overrides and prop was correct

**Review by mutebg — CHANGES_REQUESTED** [review]

**Comment by Srinivasan-Ramasamy** [comment]
Thanks @mutebg  and @baburay23 for reviewing the code :+1:

File: site/pages/components/select.tsx (hunk 1 above)
Comment (Srinivasan-Ramasamy) [reply site/pages/components/select.tsx]: Sure Stoyan, I've modified this.

File: site/pages/components/select.tsx (hunk 3 above)
Comment (Srinivasan-Ramasamy) [reply site/pages/components/select.tsx]: thanks @mutebg , I've updated the tables.

**Review by mutebg — APPROVED** [review]

**Review by baburay23 — APPROVED** [review]

**Pull request merged**